plugins/org.eclipse.che.editor.dirigible/1.0.0/meta.yaml

```diff
@@ -0,0 +1,8 @@
+id: dirigible-che-editor-plugin
```

Comment:
```suggestion
id: org.eclipse.che.editor.dirigible
```

Reply:
@skabashnyuk we don't have a naming convention yet. I wouldn't go with the Java packages naming anyway (too verbose). I have created https://github.com/eclipse/che-plugin-registry/issues/55 to discuss that.

Reply:
We have convention that path should be plugins/{id}/{version}/meta.yaml